lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -231,18 +263,41 @@ def handle(self, *args, **options):
         intervals = int(0.10 * total)
         if intervals > 100 and verbose:
             intervals = 101
+        if intervals < 1:
```

Comment:
No, is not necessary since `verbosity` is not a mandatory flag, here we are just making sure that `intervals` have a value an avoid a `ZeroDivisionError`